applications/openshift/master/file_owner_openshift_pki_cert_files/rule.yml

```diff
@@ -0,0 +1,45 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: 'Verify User Who Owns The OpenShift PKI Certificate Files'
+
+description: |-
+  {{{ describe_file_owner(file="/etc/kubernetes/static-pod-resources/.*/.*/.*/tls\.crt", owner="root") }}}
+
+rationale: |-
+  OpenShift makes use of a number of certificates as part of its operation.
+  You should verify the ownership of the directory containing the PKI
+  information and all files in that directory to maintain their integrity.
+  The directory and files should be owned by the system administrator.
```

Comment:
Nit: Other rules say owned by root:root instead of system administrator, should we say root here also?

Reply:
Gabe corrected me in another patch saying that "root:root" was too specific.

Reply:
In general, there should be not statement of who a directory or files are to be owned by at all in the rationale.

Reply:
@redhatrises got it, I'll remove it in a subsequent PR. Might want to add guidance on how to write these rationales to the docs.